ts/sagas/services/removeOldStoredServices.ts

```diff
@@ -0,0 +1,68 @@
+import * as pot from "italia-ts-commons/lib/pot";
```

Comment:
this file and function too, could be named **_removeUnusedStoredServices.ts_**
What do you think?

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1503/commits/4081a008d43c026da44da6003142b34298c558bb